prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -610,4 +610,4 @@ periodics:
+      - "-images=projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2004-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2004-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2204-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2204-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2404-lts-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-2404-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2004-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2004-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2204-lts,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2204-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2404-lts-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2404-lts-arm64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2410-amd64,projects/ubuntu-os-cloud-devel/global/images/family/ubuntu-minimal-2410-arm64"
```

Comment:
You dropped 1604 and 1804 here but left them in on cit-periodics and osogin-periodics. Do you want to keep/drop them everywhere?

Reply:
I should have just left Ubuntu 16.04 and 18.04 Pro there (not the EOL LTS ones).